bigtable-hbase-dataflow/src/test/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfigurationTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.dataflow;
+
+import org.apache.hadoop.hbase.client.Scan;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.cloud.bigtable.dataflow.CloudBigtableScanConfiguration;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+/**
+ * Tests for {@link CloudBigtableScanConfiguration}.
+ */
+public class CloudBigtableScanConfigurationTest {
+
+  @Test
+  public void testSerialization() throws IOException, ClassNotFoundException{
+    byte[] startRow = "aa".getBytes();
+    byte[] stopRow = "zz".getBytes();
+    String projectId = "project";
+    String zone = "zone";
+    String cluster = "cluster";
+    String table = "table";
+    CloudBigtableScanConfiguration config = new CloudBigtableScanConfiguration.Builder()
+      .withProjectId(projectId)
+      .withZone(zone)
+      .withCluster(cluster)
+      .withTable(table)
+      .withScan(new Scan(startRow, stopRow))
+      .build();
+
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
+    ObjectOutputStream oos = new ObjectOutputStream(bos);
```

Comment:
Look at [SerializableUtils](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/bdde37492d5f79b3570a543fe6392391023dddec/sdk/src/main/java/com/google/cloud/dataflow/sdk/util/SerializableUtils.java) for helpers to test serializability.